README.markdown

```diff
@@ -167,6 +167,67 @@ The result will be as follows:
   notice: Scope(Class[main]): FtvfvkS9j9wXLsd6
 ```
 
+This functions is not "pure" and will return different value on every call.
```

Comment:
If this is modifying the docs for the above function that should be a different commit.